file1.txt

```diff
@@ -1 +1 @@
-full feature implemented
```

Comment:
For adding password feature, you have remove previously added feature.
So, please re-add that feature.

Reply:
Yes, I will re-add that feature. That for reviewing